pkg/rancher-desktop/backend/kube/wsl.ts

```diff
@@ -188,20 +184,11 @@ export default class WSLKubernetesBackend extends events.EventEmitter implements
     }
   }
 
-  /**
-   * Install K3s into the VM for execution.
-   * @param version The version to install.
-   */
-  protected async installK3s(version: semver.SemVer) {
+  async install(config: BackendSettings, version: semver.SemVer) {
```

Comment:
is `config` being used at all?

Reply:
Nope (there's in fact two more unused arguments), it's just there to match the method signature on the interface this implements.